Makefile

```diff
@@ -123,7 +123,7 @@ deploy: dist
 ## or copying the files directly to a sibling mattermost-server directory.
 ifneq ($(and $(MM_SERVICESETTINGS_SITEURL),$(MM_ADMIN_USERNAME),$(MM_ADMIN_PASSWORD),$(CURL)),)
 	@echo "Installing plugin via API"
-	$(eval TOKEN := $(shell curl -i -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/users/login -d '{"login_id": "$(MM_ADMIN_USERNAME)", "password": "$(MM_ADMIN_PASSWORD)"}' | grep -o MMAUTHTOKEN=[0-9a-z]\* | cut -f2 -d'=' 2> /dev/null))
+	$(eval TOKEN := $(shell curl -i -X POST $(MM_SERVICESETTINGS_SITEURL)/api/v4/users/login -d '{"login_id": "$(MM_ADMIN_USERNAME)", "password": "$(MM_ADMIN_PASSWORD)"}' | grep -o "Token: [0-9a-z]*" | cut -f2 -d' ' 2> /dev/null))
```

Comment:
Should this change also be applied to the sample-plugin?

Reply:
Yes, looks like that's the new way

Reply:
Do you know when this change was introduced? Does the "old" command not work any longer?

I'm trying to figure out if we have to communicate this change.

Reply:
Frankly I was caught by surprise a little, and learned about this because my make stopped working. Definitely looks like a breaking change. @crspeller Can you answer the ^^ please?

Reply:
I think I chaned this a some point because whatever commands where being used here where not Linux compatible.

Reply:
Are the changes documented and/or reflected in the sample and demo plugins? @crspeller

Reply:
> I think I chaned this a some point because whatever commands where being used here where not Linux compatible.

@crspeller I'm not sure I get what you are saying. I thought the "old" command was compatible with Linux. I did had problems on various systems with it. What exactly was/is the issue?